setup/quickstart/halyard.md

```diff
@@ -9,4 +9,5 @@ We've written guides for a few environments/configurations to get you started
 with Spinnaker by using Halyard. If your desired environment matches one of 
 those listed below, we recommend following the linked guide. 
 
-* [Deploy Spinnaker to GKE](/guides/tutorials/codelabs/halyard-getting-started)
+* [Halyard on GKE Quickstart](/setup/quickstart/halyard-gke/)
+* [Halyard on GCE Quickstart](/setup/quickstart/halyard-gce/)
```

Comment:
Yay!

However, I think "GKE" is kind of an insider term - the product name is "Container Engine", so we should probably use that.